Be more specific about signatures in the updater (#856)
tauri-apps/tauri-docs: docs/guides/distribution/updater.md

"pub_date" if present must be formatted according to [RFC 3339][date and time on the internet: timestamps].

> "signature" is content of .sig file that was generated by Tauri CLI while using keys that were generated by Tauri CLI [See Signing Updates](#signing-updates)

Reviewer: ```suggestion
"signature" is the content of the `.sig` file which was generated by Tauri's CLI. See [Signing Updates](#signing-updates) for instructions on how to set up the required keys.
```

The Tauri bundler automatically generates update artifacts if the updater is enabled in `tauri.conf.json`
Your update artifacts are automatically signed if the bundler can locate your private and public key.

> The signature can be found in the content of a `sig` file. The signature can be uploaded to GitHub safely or made public if your private key is secure.

Reviewer: ```suggestion
The signature is the content of the generated `.sig` file. The signature can be uploaded to GitHub safely or made public if your private key is secure.
```
